### PR TITLE
Added te.po - Telugu Translation

### PR DESCRIPTION
**Pull request by SpaciousCoder78**
Added telugu translation

### CODE REVIEW

File: po/te.po [hunk 1]
```diff
@@ -0,0 +1,2687 @@
...
+msgid "Fixed spinner not hiding if the back end fails"
+msgstr "బ్యాక్ ఎండ్ విఫలమైతే దాక్కోని స్థిర స్పిన్నర్"
+
+#: data/com.jeffser.Alpaca.metainfo.xml.in:87
+msgid "Fixed image recognition with local images"
+msgstr "స్థానిక చిత్రాలతో స్థిర చిత్రం గుర్తింపు"
+
+#: data/com.jeffser.Alpaca.metainfo.xml.in:88
+msgid "Changed appearance of delete / stop model buttons"
+msgstr "డిలీట్/స్టాప్ మోడల్ బటన్‌ల రూపాన్ని మార్చారు"
```
Comment (olumolu) [inline po/te.po:146]: Add space / before and after

**Review by olumolu — CHANGES_REQUESTED** [review]

File: po/te.po [hunk 2]
```diff
@@ -0,0 +1,2687 @@
...
+msgid "Restored sidebar toggle button"
+msgstr "సైడ్‌బార్ టోగుల్ బటన్ పునరుద్ధరించబడింది"
+
+#: data/com.jeffser.Alpaca.metainfo.xml.in:232
+msgid "Reverted back to standard styles"
+msgstr "ప్రామాణిక శైలులకు తిరిగి మార్చబడింది"
+
+#: data/com.jeffser.Alpaca.metainfo.xml.in:233
+msgid "Fixed generated titles having \"'S\" for some reason"
+msgstr "కొన్ని కారణాల వల్ల \"'S\"ని కలిగి ఉన్న స్థిర జనరేట్ శీర్షికలు"
```
Comment (olumolu) [inline po/te.po:445]: \"'S\"_ space needed \"'S\"ని

File: po/te.po (hunk 1 above)
Comment (SpaciousCoder78) [reply po/te.po]: Fixed

File: po/te.po (hunk 2 above)
Comment (SpaciousCoder78) [reply po/te.po]: Fixed

**Comment by SpaciousCoder78** [comment]
Updated the translation with suggested changes

**Comment by Jeffser** [comment]
That's fine, thanks for your contribution!

**Comment by SpaciousCoder78** [comment]
Welcome

**Pull request merged**